hedera-node/hedera-app/src/main/java/com/hedera/node/app/ServicesMain.java

```diff
@@ -375,7 +393,9 @@ public static Hedera newHedera(@NonNull final NodeId selfNodeId, @NonNull final
     public static Configuration buildPlatformConfig() {
         final ConfigurationBuilder configurationBuilder = ConfigurationBuilder.create()
                 .withSource(SystemEnvironmentConfigSource.getInstance())
-                .withSource(SystemPropertiesConfigSource.getInstance());
+                .withSource(SystemPropertiesConfigSource.getInstance())
+                .withConfigDataType(BasicConfig.class);
```

Comment:
not sure if this is needed.

`BootstrapUtils.setupConfigBuilder` uses `autoDiscoverExtensions()`